lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Tab = props => {
-  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
-      if they match, the className should be: 'tab active-tab', 
-      if it is not it should just be 'tab'*/
+if (props.tab)
   return (
     <div
-      className={''}
-      onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
-      }}
-    >
+    className={
+      props.selected === props.selectedTab ? 'tab active-tab' : 'tab'
```

Comment:
close, but this should be `props.tab === props.selectedTab` as each tab checks to see if it's tab prop is the same as the selectedTab prop that is passed from from Content.js